packages/protocol-utils/package.json

```diff
@@ -19,8 +19,8 @@
         "test:coverage": "mkdir -p ./coverage && nyc npm run test --all && yarn coverage:report:lcov",
         "coverage:report:lcov": "nyc report --reporter=text-lcov > coverage/lcov.info",
         "clean": "shx rm -rf lib generated_docs",
-        "lint": "tslint --format stylish --project .",
-        "fix": "tslint --fix --format stylish --project .",
+        "lint": "",
```

Comment:
we don't need to fix all warnings 
`eslint --quiet` (ignores warnings) would be a good start

Reply:
or `eslint --max-warnings N`


https://github.com/0xProject/protocol/commit/76c0ce2b2a9e5e5a02c373df6d5d80b9ffb2e7d5

Reply:
I set it as eslint --quiet. Let me know if this looks good!

Reply:
Either works!